WordPress/Classes/ViewRelated/Post/PostEditor+MoreOptions.swift

```diff
@@ -76,6 +76,8 @@ extension PostEditor {
             return
         }
 
+        emitPostSaveEvent()
```

Comment:
I initially attempted to call this within the `savePostBeforePreview` function, but this resulted in the event being sent too late.